packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -301,6 +327,27 @@ class LinearProgressIndicator extends ProgressIndicator {
   /// {@endtemplate}
   final double? minHeight;
 
+  /// The border radius of the widget.
```

Comment:
From the docs I am not sure what this value is for.

Reply:
What do you think?
<img width="692" alt="image" src="https://user-images.githubusercontent.com/351125/232667260-19e8101b-95cd-42bf-a455-2d5fc4178af1.png">